quickwit/quickwit-indexing/src/lib.rs

```diff
@@ -70,13 +71,15 @@ pub async fn start_indexing_service(
     num_blocking_threads: usize,
     cluster: Cluster,
     metastore: MetastoreServiceClient,
-    ingest_api_service: Mailbox<IngestApiService>,
```

Comment:
This is going away :) BTW, I claim the removal of the ingest v1 code :D

Reply:
have a blast!